zeebe/dynamic-config/src/main/java/io/camunda/zeebe/dynamic/config/changes/PartitionDeleteExporterApplier.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.dynamic.config.changes;
+
+import io.atomix.cluster.MemberId;
+import io.camunda.zeebe.dynamic.config.changes.ConfigurationChangeAppliers.MemberOperationApplier;
+import io.camunda.zeebe.dynamic.config.state.ClusterConfiguration;
+import io.camunda.zeebe.dynamic.config.state.DynamicPartitionConfig;
+import io.camunda.zeebe.dynamic.config.state.MemberState;
+import io.camunda.zeebe.scheduler.future.ActorFuture;
+import io.camunda.zeebe.scheduler.future.CompletableActorFuture;
+import io.camunda.zeebe.util.Either;
+import java.util.function.UnaryOperator;
+
+final class PartitionDeleteExporterApplier implements MemberOperationApplier {
```

Comment:
:x: We need a test for this class that verifies following behaviors:

* Init fails when
    - If exporter state is not `CONFIG_NOT_FOUND`
    - no member exists
    - no partition exists
    - no exporter exists
* Apply returns failed future when `partitionChangeExecutor` fails
* Success scenario: The exporter does not exist in config, after the operation is applied